doc/devices/numpy_wavefunction.rst

```diff
@@ -0,0 +1,73 @@
+The Numpy-Wavefunction device
+=============================
+
+The ``forest.numpy_wavefunction`` device provides an interface between PennyLane
+and the pyQuil ``NumpyWavefunctionSimulator``.
+
+As the NumPy wavefunction simulator allows access and manipulation of the underlying
+quantum state vector, ``forest.numpy_wavefunction`` is able to support the full
+suite of PennyLane and Quil quantum operations and observables.
+
+
+.. note::
+
+    Since the NumPy wavefunction simulator is written entirely in NumPy, no external
+    Quil compiler is required.
+
+
```

Comment:
```suggestion
    Since the NumPy wavefunction simulator is written entirely in NumPy, no external
    Quil compiler is required.

```

A trivial change to force a PR re-build.